types/releases.ts

```diff
@@ -0,0 +1,37 @@
+// TODO(HinataKah0): Remove this, see https://github.com/nodejs/nodejs.org/issues/5303#issuecomment-1518947513
+/* eslint-disable no-unused-vars */
```

Comment:
Do we really need this eslint ignore?

Reply:
Yes, if we define the enum values, it will upset ESLint because they aren't used.

Reply:
instead of enum, can we try

```js
const ReleaseTypes = {
current: 'Current',
  lts: 'LTS'
} as const
```

eslint wouldn't bother this

Reply:
I also think we could just make a TypeScript type instead of an enum, no?

Reply:
Yes, it also works. I've tested it and ESLint isn't upset.
I read [this](https://sergiocarracedo.es/typescript-enums/). It looks like Typescript doc also recommends using `as const`.
Thanks for the input!
```diff
diff --git a/types/releases.ts b/types/releases.ts
index 86766f77..1666d7d7 100644
--- a/types/releases.ts
+++ b/types/releases.ts
@@ -1,19 +1,24 @@
-// TODO(HinataKah0): Remove this, see https://github.com/nodejs/nodejs.org/issues/5303#issuecomment-1518947513
-/* eslint-disable no-unused-vars */
+export const ReleaseTypes = {
+  current: 'Current',
+  lts: 'LTS',
+  maintenance: 'Maintenance',
+  endOfLife: 'End-of-life',
+} as const;
 
-export enum ReleaseTypes {
-  current = 'Current',
-  lts = 'LTS',
-  maintenance = 'Maintenance',
-  endoflife = 'End-of-life',
-}
+type ReleaseType = (typeof ReleaseTypes)[keyof typeof ReleaseTypes];
 
 export interface UpcomingReleaseData {
   releaseDate: string;
-  releaseType: ReleaseTypes;
+  releaseType: ReleaseType;
   alreadyReleased: boolean;
 }
 
+let testNoError = {
+  releaseDate: '',
+  releaseType: ReleaseTypes.current,
+  alreadyReleased: true,
+} as UpcomingReleaseData;
+
 export interface UpcomingRelease {
   title: string;
   releases: UpcomingReleaseData[];
```

Reply:
I think what I'd suggest is to not create a variable at all. Just a TypeScript type.

E.g., we don't need an enumerator/variable for this. Just a type.

So if in the past you had something like:

```tsx
type Props = {
 releaseType: ReleaseTypes // this is enum
}

// usage of component
<ReleaseWhatever releaseType={ReleaseTypes.LTS} />
```

you would have the following with a plain type:

```tsx
type ReleaseType = 'LTS' | 'Current' | ...

type Props = {
 releaseType: ReleaseType // this is a type
}

<ReleaseWhatever releaseType="LTS" />
```

Reply:
Ah, missed this 🤯 
I searched and saw it is used by others as well, thanks!

Reply:
Feel free to resolve this conversation once you've made these changes :)